WordPress/FEATURE_ANNOUNCEMENTS.json-example

```diff
@@ -0,0 +1,28 @@
+// remove comments from actual announcements
```

Comment:
wdyt if we add in comment also the asset expected target folder?

I was also thinking to a way to avoid the need to remove the comments from the template, the only thing I could think of is to include extra fields like:

```
"_template_note": "app version name that announcement targets. Used for display purposes only.",
"appVersionName": "14.7", 
```

This would be good to avoid the need of removing comments but it's also not great since it introduces extra fields and maybe can overcomplicate things with GSON (or even making it not scalable?). Not feeling strong about it so let me know if you think it's worth or you have a better fit, otherwise I'm fine with keeping the comments 😊

Reply:
Good idea on including file location 👍 

You actually don't need to remove comments, GSON is smart enough to ignore them :) I also don't have a strong opinion on this. Either way, I think that people probably going to remove comments.